src/styles/nearByWhoLikedMeStyles.ts

```diff
@@ -1,58 +1,102 @@
 import { makeStyles } from 'tss-react/mui'
-export const nearByWhoLikedMeStyles = makeStyles()(() => {
+
+export const nearByWhoLikedMeStyles = makeStyles()((theme) => {
   return {
```

Comment:
Please delete return

Reply:
You wrote `useStyles` instead of `nearByWhoLikedMeStyles` and deleted export. That's why you got the errors.
In this case it works:
![image](https://github.com/WeFriiends/WeFriiendsWeb/assets/81742845/889a50f6-e03e-4fa9-8936-e5caacff2d6d)

Reply:
Done.